docs/section22/Rule22-5.md

```diff
@@ -0,0 +1,59 @@
+
+# CHW&CW - Rule 22-5  
+
+**Rule ID:** 22-5  
+**Rule Description:** For Baseline chilled water loop that is not purchased chilled water and serves computer room HVAC systems (System Type-11), chilled-water supply temperature shall be reset higher based on the HVAC system requiring the most cooling.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system serves computer room HVAC system (System Type-11)
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule 22-3.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system serves computer room HVAC system (System Type-11):
+ `LEN(hvac.does_serve_computer_room == TRUE for hvac in B-RMR...heating_ventilation_air_conditioning_systems) > 0`
+
```

Comment:
similar comment as the one made in 22-4, need to  find zones associated with HVAC system, spaces associated with the zone, space type of each space and then identify if it is a computer room.

Reply:
I think this is already determined by the RDS that confirmed that there is a System 11 on the project. I don't think we need to check it again.

Reply:
@mkarpman sure, then this applicability check should be changed to 'does the B RMR has a HVAC system type 11.'